content/applications/hr/employees/new_employee.rst

```diff
@@ -295,55 +306,53 @@ however, some information in this section may be critical for the company's payr
 order to properly process payslips and ensure all deductions are accounted for, the employee's
 personal information should be entered.
 
-Here, the employee's :guilabel:`Private Contact`, :guilabel:`Education`, :guilabel:`Work Permit`,
-:guilabel:`Family Status`, :guilabel:`Emergency` contact, and :guilabel:`Citizenship` information is
+Here, the employee's :guilabel:`Private Contact`, :guilabel:`Family Status`, :guilabel:`Emergency`
+contact, :guilabel:`Education`, :guilabel:`Work Permit`, and :guilabel:`Citizenship` information is
 entered. Fields are entered either using a drop-down menu, activating a check box, or typing in the
 information.
 
-- :guilabel:`Private Contact`: Enter the personal :guilabel:`Address` for the employee. The
-  selection can be made with the drop-down menu. If the information is not available, type in the
-  name for the new address. To edit the new address, click the :guilabel:`Internal Link` arrow to
-  open the address form. On the address form, enter or edit the necessary details. Some other
-  information in the :guilabel:`Private Contact` section may auto-populate, if the address is
-  already listed in the drop-down menu.
-
-  Enter the employee's :guilabel:`Email` address and :guilabel:`Phone` number in the respective
-  fields. Next, enter the employee's :guilabel:`Bank Account Number` using the drop-down menu. If
-  the bank is not already configured (the typical situation when creating a new employee) enter the
-  bank account number, and click :guilabel:`Create and edit..`. A :guilabel:`Create Bank Account
-  Number` form loads. Fill in the information, then click :guilabel:`Save & Close`.
-
-  Then, select the employee's preferred :guilabel:`Language` from the drop-down menu. Next, enter
-  the :guilabel:`Home-Work Distance` in the field. This field is only necessary if the employee is
+- :guilabel:`Private Contact`: first, enter the :guilabel:`Private Address` for the employee. Next,
+  enter the employee's :guilabel:`Email` address and :guilabel:`Phone` number in the respective
+  fields. Then enter the employee's :guilabel:`Bank Account Number` using the drop-down menu. If the
+  bank is not already configured (the typical situation when creating a new employee) enter the bank
+  account number, and click :guilabel:`Create and edit..`. A :guilabel:`Create Bank Account Number`
+  form loads. Fill in the information, then click :guilabel:`Save & Close`. Next, select the
+  employee's preferred :guilabel:`Language` from the drop-down menu. Then enter the
+  :guilabel:`Home-Work Distance` in the field. This field is only necessary if the employee is
```

Comment:
optional rewording suggestions and starting new paragraph when addressing a new subfield. Totally up to your own personal writing style!
```suggestion
- :guilabel:`Private Contact`: Enter the :guilabel:`Private Address`, :guilabel:`Email`, and
  :guilabel:`Phone` for the employee. Then enter the employee's :guilabel:`Bank Account Number`
  using the drop-down menu.

  If the bank is not configured, enter the bank account number, and click :guilabel:`Create and
  edit..`. In the :guilabel:`Create Bank Account Number` form, fill in the information, and click
  :guilabel:`Save & Close`.

  Next, select the employee's preferred :guilabel:`Language` from the drop-down menu. Then enter the
  :guilabel:`Home-Work Distance` in the field. This field is only necessary if the employee is
```